api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -342,9 +342,8 @@
     <concept  concept_id="163275"  retired="0" datatype_id="4"  class_id="5"  is_set="0"  creator="1"  date_created="2016-02-20 19:32:10"  version="NULL"  changed_by="1"  date_changed="2016-02-20 19:38:44"  retired_by="1" uuid="163275AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
     <concept  concept_id="163290"  retired="0" datatype_id="4"  class_id="5"  is_set="0"  creator="1"  date_created="2016-02-28 21:15:28"  version="NULL"  changed_by="1"  date_changed="2016-02-28 21:16:03"  retired_by="1" uuid="163290AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
     <concept  concept_id="163291"  retired="0" datatype_id="4"  class_id="5"  is_set="0"  creator="1"  date_created="2016-02-28 21:17:17"  version="NULL"  changed_by="1"  date_changed="2016-02-28 21:17:26"  retired_by="1" uuid="163291AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-    <concept  concept_id="163399"  retired="0" datatype_id="4"  class_id="10"  is_set="1"  creator="1"  date_created="2016-06-01 21:21:42"  version="NULL"  changed_by="1"  date_changed="2016-07-01 16:01:09"  retired_by="1" uuid="163399AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
@slubwama Why delete this Common Occupations?

Reply:
@ssmusoke its a conv set yet there is another one for occupation which is being used. this is not used. It was not supposed to be created in the first place